swap_meet/clothing.py

```diff
@@ -1,2 +1,11 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+class Clothing(Item):
+    def __init__(self, condition = 0, category = "Clothing" ):
```

Comment:
It would be a good idea to remove category from the `__init__` function. We don't want the user to be able to set the category. It should always be "Clothing" for the Clothing class. Instead, you can just add "Clothing" as the category when you call super.